<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>694</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1045</width>
    <height>694</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1045</width>
    <height>694</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vista Corso</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>694</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>VisualizzaIscrittiCorso.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>431</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="list_view">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_inserisciIscritti">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>INSERISCI ISCRITTI CORSO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_rimuoviIscritti">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>RIMUOVI ISCRITTI CORSO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>113</y>
     <width>260</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightgrey;
border-radius: 15px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeroIscritti">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>162</y>
     <width>81</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightgrey;
border-radius: 15px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>button_inserisciIscritti</zorder>
  <zorder>tipo</zorder>
  <zorder>numeroIscritti</zorder>
  <zorder>label_2</zorder>
  <zorder>button_rimuoviIscritti</zorder>
 </widget>
 <resources>
  <include location="VisualizzaIscrittiCorso.qrc"/>
 </resources>
 <connections/>
</ui>
